control/pzmap.py

```diff
@@ -40,11 +40,42 @@
 #
 # $Id:pzmap.py 819 2009-05-29 21:28:07Z murray $
 
-from numpy import real, imag
-from .lti import LTI
+from numpy import real, imag, linspace, pi, exp, cos, sin, sqrt
```

Comment:
Rather than using numpy.pi, please use math.pi to avoid problems with sphinx.  See commit aa6e0dfc05fde5e70d48f1add3f89e01adb1e8d5

Reply:
ok